src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -99,7 +99,11 @@ def assertRegConditionalsForPpaRefModel(registration_requests,
           # If the cbsdCategory is not present in registration request then
           # assign it to the cbsdCategory in conditional_params.
           if 'cbsdCategory' not in device:
-            break
+            if not conditional_params.get('cbsdCategory'):
```

Comment:
If you just do `conditional_params['cbsdCategory']`, this will automatically raise a KeyError exception. So you can make this a lot simpler :)

Reply:
Simpler is better. :) Submitted. Thanks, Kate!